pycraigslist/query/filters.py

```diff
@@ -55,7 +56,7 @@ def parse_filters(filters, query_filters, **kwargs):
             try:
                 filters[key] = [query_filters[key][parsed_val] for parsed_val in parse_value(value)]
             except (KeyError, TypeError):
-                raise ValueError("filter '%s' is or has a bad value" % key)
+                raise InvalidFilterValue("filter '%s' is or has a bad value" % key, key, value)
```

Comment:
If we remove the constructor method in `InvalidFilterValue`, we should remove the terminal `key, value` args.